### PR TITLE
added MGnify Search tutorial and supporting data

### PR DESCRIPTION
**Pull request by ellihung**

### CODE REVIEW

**Review by ababaian — COMMENTED** [review]
Heya thanks for the PR : )

Can I request a few changes.

1. change the name of your tutorial to `tools/mgnify_search.md` to be uniform with the rest of the pages

File: tools/MGnify_search.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  * Human - non-digestive
+  * Animal
+
+* **Other (sequences not found in other environment or biome categories)**
+  * Engineered
+  * Other
+  
+In this example, we will search only for full length sequences.
+
+![](img/MGnify_search/images/sequence_subset.png)
```
Comment (ababaian) [inline tools/MGnify_search.md]: Add an image sub-title here, kind of like a figure legend.

File: tools/MGnify_search.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+### 4. Result page
+
+A list of matching sequences is presented that is ordered by E-value significance (lower E-value indicating a more significant match). The user can customize this table to dispaly different headings. Clicking on the `Target` link shows the amino acid FASTQ file that the query sequence matched to. the `Run & Sample IDs` link brings up the associated MGnify Sample Overview.
+
+In our [example](https://www.ebi.ac.uk/metagenomics/sequence-search/results/7CFA9DC8-900B-11EE-B07A-5352765DAC3F/score), 2368 significant query matches were found. The top match has a bit score of 890.8 and an E-value of 3.5e-265.
+
+![](img/MGnify_search/images/results.png)
```
Comment (ababaian) [inline tools/MGnify_search.md]: ditto here

**Comment by ababaian** [comment]
Likewise for the images files, change the directory to `mgnify_search/` so everything is nice and uniform. A couple more comments for figure titles and you're good :+1:

**Comment by ellihung** [comment]
Hi Artem,

Thanks for this! I changed the items based on your comments and have pushed to GitHub. Do I need to do another pull request?

Thanks,
Elli

**Comment by ababaian** [comment]
The changes should show up here.

File: tools/MGnify_search.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+![database options under sequence subset](img/mgnify_search/sequence_subset.png)
+
+### 4. Result page
+
+A list of matching sequences is presented that is ordered by E-value significance (lower E-value indicating a more significant match). The user can customize this table to dispaly different headings. Clicking on the `Target` link shows the amino acid FASTQ file that the query sequence matched to. the `Run & Sample IDs` link brings up the associated MGnify Sample Overview.
+
+In our [example](https://www.ebi.ac.uk/metagenomics/sequence-search/results/7CFA9DC8-900B-11EE-B07A-5352765DAC3F/score), 2368 significant query matches were found. The top match has a bit score of 890.8 and an E-value of 3.5e-265.
+
+![](img/mgnify_search/results.png)
```
Comment (ababaian) [inline tools/MGnify_search.md]: One more title link change here.

**Review by ababaian — COMMENTED** [review]
you one link

**Review by ababaian — APPROVED** [review]

**Comment by ellihung** [comment]
Hi Artem,

Let me know if I did that correctly… I updated things that never showed up on GitHub and got confused and I hope I didn’t accidentally do something to mess it up for others. I’m still not seeing the name change from MGnify_search.md to mgnify_search.md on GitHub even though that’s what it’s showing on my local repo.

Thanks,
Elli

**Comment by ababaian** [comment]
Oh that's actually my bad, I thought I caught the change. No worries I'm going to do a little bit of parsing of all the new pull requests so I can amend it then. Thanks for the changes! Enjoy the weekend.

**Pull request merged**